docs/changes.rst

```diff
@@ -32,27 +32,16 @@ Released: not yet
 * Installation of this package using "setup.py" is no longer supported.
   Use "pip" instead.
 
-* Update to support urllib3 package version 2.0+ (for python versions 3.7+)
+* Update to support python urllib3 dependent package version 2.0+ (for python versions 3.7+)
```

Comment:
pywbem 1.6.0 did not yet pin urllib3 to <2.0. So the change in 1.7.0 w.r.t. urllib3 is to **introduce** a pinning of urllib3 to <2.0 for Python <=3.6. This change log entry does not correctly reflect that change.

Beyond that, the pinning of urllib3 to <2.0 in the current master branch really means that we are not supporting the use of Python<=3.6 with OpenSSL 1.1.1 (which seems to be a possible combination) or with the supported LibreSSL versions. I don't understand what would cause that restriction, and therefore I think that pinning of urllib3 to <2.0 on Python <=3.6 is incorrect and should be removed. Instead, we should exclude OpenSSL versions 2.0.0 - 2.0.2 since they do not support LibreSSL, at least on macos. So the requirements.txt file should become:
```
urllib3>=1.26.18; sys_platform != 'darwin'
urllib3>=1.26.18,!=2.0.0,!=2.0.1,!=2.0.2; sys_platform == 'darwin'
```
and the minimum-constraints.txt file:
```
urllib3==1.26.18
```
In that case, this change log entry would need to be adjusted.

Let's discuss this.

Reply:
Actually we did not really support urllib3 2.0+ because we released 1.6 without testing against that version. 

We pinned urllib3 to < 2.0 because the specifically stated that they do not support python 2.7, 3.5 and 3.6 with this new version.

We also made some changes although they were really in the just cim_http module and tests because of some exception issues. 

In any case, I agree that the whole statement is not clear since the change is really to limit the urllib3 version for Python <= 3.6.  The secondary effect was for us to specifically test with urllib3 version 2.0+ and determine